vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -244,6 +244,20 @@ void UwpReactInstance::Start(const std::shared_ptr<IReactInstance>& spThis, cons
     devSettings->useJITCompilation = settings.EnableJITCompilation;
     devSettings->debugHost = settings.DebugHost;
 
+    // In most cases, using the hardcoded ms-appx URI works fine, but there are certain scenarios,
+    // such as in optional packaging, where the developer might need to modify the path, in which
+    // case we should use the custom path instead.
+    if (settings.BundleRootPath.empty())
```

Comment:
I'm not sure how others feel but I prefer to use a single assignment with conditional (? :) in a case like this.

```
devSettings->bundleRootPath = settings.BundleRootPath.empty() ? "ms-appx:///Bundle/" : settings.BundleRootPath;
```

Reply:
Done.